docs/siem/detections/api/rules-api-create.asciidoc

```diff
@@ -60,145 +65,235 @@ calls, see:
+
+|==============================================
+
+===== Required field for threshold rules
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|threshold |Object a|Defines the field and threshold value for when alerts
+are generated, where:
+
+* `field` (string, required): The field on which the threshold is applied. If
+you specify an empty field (`""`), alerts are generated when the query returns
+at least the number of results specified in the `value` field.
+* `value` (integer, required): The threshold value from which an alert is
+generated.
+
+|==============================================
+
+===== Required field for saved-query rules
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|saved_id |String |Kibana saved search used by the rule to create alerts.
+
+|==============================================
+
+===== Required fields for machine-learning rules
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|anomaly_threshold |Integer |Anomaly score threshold above which the rule
+creates an alert. Valid values are from `0` to `100`.
 
 |machine_learning_job_id |String |{ml-cap} job ID the rule monitors for
-anomaly scores. |Yes, for {ml} rules. Not allowed in `query` rules types.
+anomaly scores. 
 
-|query |String |{kibana-ref}/search.html[Query] used by the rule to create a 
-signal. |No. For `query` rules types, defaults to an empty string. Not allowed
-in `machine-learning` rule types.
+|==============================================
 
-|language |String |Determines the query language, which must be
-`kuery` or `lucene`. |No. For `query` rule types, defaults to `kuery`. Not
-allowed in `machine-learning` rule types.
+===== Optional fields for all rule types
 
-|output_index |String |Index to which signals detected by the rule are saved. 
-|No, if unspecified signals are saved to `.siem-signals-<space_name>` index, 
-where `<space_name>` is the name of the {kib} space in which the rule exists.
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
 
-|saved_id |String |Kibana saved search used by the rule to create signals.
-|Yes, for `saved_query` rules only.
+|actions |<<actions-object-schema, actions[]>> |Array defining the automated
+actions (notifications) taken when alerts are generated.
 
-|meta |Object |Placeholder for metadata about the rule. |No
+|author |String[] |The rule's author.
 
-|risk_score |Integer a|A numerical representation of the signal's severity from 
-0 to 100, where:
+|building_block_type |String |Determines if the rule acts as a building block.
+By default, building-block alerts are not displayed in the UI. These rules are
+used as a foundation for other rules that do generate alerts. Its value must be
+`default`. For more information, see BEN - add link later.
```

Comment:
@benskelker - I was chatting with @MikePaquette and he said he may have a good example or two for us to reference when speaking about Building Block Rules, so stay tuned 🙂

Reply:
Much needed :)